### PR TITLE
Show team on stats -  #170

### PR DESCRIPTION
**Pull request by Nilint1**
Adds enhancement:  #170 
Team page:
image

Players page:
image

### CODE REVIEW

File: components/ScoreTable/GoalieScoreTable.tsx [hunk 1]
```diff
@@ -6,11 +6,13 @@ import { Goalie } from '../..';
 interface Props {
   data: Array<Goalie>;
   pagination?: boolean;
+  teamPage?: boolean;
```
Comment (Nilint1) [inline components/ScoreTable/GoalieScoreTable.tsx:9]: added teamPage flag to props for score tables

File: components/ScoreTable/GoalieScoreTable.tsx [hunk 2]
```diff
@@ -23,6 +25,11 @@ function GoalieScoreTable({
           accessor: 'name',
           // Create cell which contains link to player
         },
+        {
```
Comment (Nilint1) [inline components/ScoreTable/GoalieScoreTable.tsx:28]: add team to component

File: components/ScoreTable/GoalieScoreTable.tsx [hunk 3]
```diff
@@ -104,6 +111,7 @@ function GoalieScoreTable({
       data={players}
       columnData={columnData}
       pagination={pagination}
+      teamPage={teamPage}
```
Comment (Nilint1) [inline components/ScoreTable/GoalieScoreTable.tsx:114]: pass flag up

File: components/ScoreTable/index.tsx [hunk 4]
```diff
@@ -22,20 +22,36 @@ interface Props {
...
   columnData,
+  teamPage = false,
 }: // isLoading
 Props): JSX.Element {
   // ! add loading state
   const data = useMemo(() => players, [players]);
 
   // ! add loading state
-  const columns = useMemo(() => columnData, []);
+  const columns = useMemo(() => {
```
Comment (Nilint1) [inline components/ScoreTable/index.tsx:40]: this is the logic that does all the magic.  Essentially, if a teamPage is used, loop through the columns of the first section.  If the header equals Team, remove it.

File: pages/[league]/team/[teamid].tsx [hunk 5]
```diff
@@ -142,9 +142,9 @@ function TeamPage({
           {!isLoading && (
             <TableContainer>
               {display ? (
-                <SkaterScoreTable data={getSkaters()} />
+                <SkaterScoreTable data={getSkaters()} teamPage />
```
Comment (Nilint1) [inline pages/[league]/team/[teamid].tsx:145]: add flag on team pages

**Comment by Nilint1** [comment]
not quite sure how to link the issue for #170

**Review by esilverm — APPROVED** [review]

**Pull request merged**